ide-support/revsblibrary.livecodescript

```diff
@@ -4,18 +4,16 @@
 #?author
 # Marcus van Houdt
 
```

Comment:
😫 whitespace diffs that I didn't notice. I'll see what I can do to clean this up.